ci/prow/config.yaml

```diff
@@ -1695,9 +1629,8 @@ periodics:
       - "--artifacts=$(ARTIFACTS)"
       - "--profile-name=coverage_profile.txt"
       - "--cov-target=."
-      - "--cov-threshold-percentage=80"
-
-- cron: "50 * * * *" # Run every hour and 50 minutes
+      - "--cov-threshold-percentage=50"
```

Comment:
coverage threshold of 50 instead of 80, is this intended?

Reply:
DItto.